content/docs/command-reference/list-url.md

```diff
@@ -0,0 +1,97 @@
+# list-url
+
+> Aliased to `dvc ls-url`
+
+List contents from a supported URL (for example `s3://`, `ssh://`, and other
+protocols).
+
+> Useful to find data to `dvc get-url` or `dvc import-url`.
+
+## Synopsis
+
+```usage
+usage: dvc ls-url [-h] [-q | -v] [-R] url
+
+positional arguments:
+  url              See `dvc get-url` for full list of supported URLs.
+```
+
+## Description
+
+Lists files and directories from an external location. `dvc ls-url` is
+equivalent to a wrapper providing a uniform interface around various listing
+commands such as `aws s3 ls` or `ssh user@host ls -a`.
```

Comment:
Minor, not a strong opinion but:
```suggestion
Lists files and directories from an external location. `dvc ls-url` is
a wrapper providing a uniform interface around various equivalent listing
commands such as `aws s3 ls` or `ssh user@host ls -a`.
```

Reply:
It doesn't actually wrap any of these commands, it just behaves similarly, hence the clunky "equivalent to a wrapper". I guess I should rephrase the whole sentence.